bitnami/airflow/values.yaml

```diff
@@ -1051,6 +1051,15 @@ metrics:
     scrapeTimeout: 10s
     # ServiceMonitor Labels
     labels: {}
+  ## @param metrics.extraEnvVars Array containing extra environment variables
+  ##
+  extraEnvVars: []
+  ## @param metrics.extraEnvVarsCM ConfigMap containing extra environment variables
+  ##
+  extraEnvVarsCM: ""
+  ## @param metrics.extraEnvVarsSecret Secret containing extra environment variables (in case of sensitive data)
```

Comment:
The descriptions are tied with the ones in the `README.md`. We have a tool that is used to sync them from time to time, we just havent completed the integration with GitHub yet.

To avoid it causing any changes to the README.md afterwards, could you make the following changes so that it doesn't overwrite some of these changes afterwards?

* Update the descriptions so they match
* Update either `README.md` or `values.yaml` so that they are added in the same location. E.g. in both files after `metrics.resources`.

Reply:
No problem, makes sense.  I probably copied the readme from web and values from scheduler or something like that.